cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -43,16 +40,35 @@ CREATE INDEX IF NOT EXISTS geom_index
 CREATE TABLE IF NOT EXISTS water_systems(
     pws_id varchar(255) NOT NULL,
     lead_connections_count real,
+    geom GEOMETRY(Geometry, 4326),
     PRIMARY KEY(pws_id)
     );
 
-ALTER TABLE water_systems
-    ADD COLUMN IF NOT EXISTS geom GEOMETRY(Geometry, 4326);
-
 CREATE INDEX IF NOT EXISTS geom_index
     ON water_systems
     USING GIST (geom);
 
+-- EPA violations data
+
+CREATE TABLE IF NOT EXISTS epa_violations(
+    violation_id varchar(255) NOT NULL,
+    violation_code varchar(255) NOT NULL,
+    compliance_status varchar(255) NOT NULL,
+    start_date DATE NOT NULL,
+    pws_id varchar(255) NOT NULL,
+    geom GEOMETRY(Geometry, 4326),
+    PRIMARY KEY(violation_id)
+    );
+
+CREATE INDEX IF NOT EXISTS geom_index
+    ON epa_violations
+    USING GIST (geom);
+
+-- Ensure faster reads on PWS id --
+CREATE INDEX IF NOT EXISTS pws_id_index
```

Comment:
Can you make this a [foreign key](https://www.postgresql.org/docs/current/tutorial-fk.html) with the `pws_id` in `water_systems`?

Reply:
Yeah for sure